### ISSUE TITLE
[BITV] Add always visible description for `NcRichContenteditable`

### ISSUE THREAD

**Issue by susnux** [nextcloud-libraries/nextcloud-vue#4429]
* As a follow up on https://github.com/nextcloud-libraries/nextcloud-vue/pull/4407

There should be a hint below the input element which is referenced using `aria-describedby` instead of using `aria-placeholder` or `placeholder`, see https://github.com/nextcloud-libraries/nextcloud-vue/pull/4407#issuecomment-1672995206

**Comment by marcoambrosini** [comment]
I don't think this should be always visible. There are place where we might not want it like in the talk new message form

**Comment by ShGKme** [comment]
Attempt was made in https://github.com/nextcloud-libraries/nextcloud-vue/pull/4482

It was decided to fix it in place in server comments: https://github.com/nextcloud/server/pull/40294

Closed now as not needed for now.
